db/seeds.rb

```diff
@@ -5,6 +5,8 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+OidMinterService.initialize_sequence!
+puts "Oid Minter Initialized"
```

Comment:
Would it be helpful to output what number the Oid Minter initialized at? Seems like a good first-step sanity check.

Reply:
Thanks, looks good.